about.md

```diff
@@ -1,91 +1,162 @@
-
- <img src="{{ site.baseurl }}/public/OSCAR-Organigramm.svg" alt="OSCAR" width="95%" style="margin-left:2%;" align="center">
-
-## The four cornerstones
-
-This project builds on four leading computer algebra systems which
-are, with the exception of the new system ANTIC, widely used
-internationally. Although the new computer algebra system tightly integrates
-the four cornerstones, the individual
-systems will continue to be developed further. This includes working
-together with the developers of the individual cornerstones
-to develop the cornerstones in a way the best integration can be achieved.
-
-[Nemo](https://github.com/Nemocas/Nemo.jl), [Hecke](http://github.com/thofma/Hecke.jl)
-are evolving number theoretic software projects focusing on
-computations in and with number fields and generic finitely presented
-rings. ANTIC is written in a mixture of C and Julia, building on top
-of the very successful [FLINT](http://flintlib.org) project of William Hart, while
-Hecke is pure Julia. In order to
-reach the mathematical goals of the OSCAR project, the functionality
-of both will be considerably extended.
-
-
-[GAP](https://www.gap-system.org) is a system for computational discrete algebra, with particular
-emphasis on group and representation theory. One of the distinguishing
-features of GAP is that it provides a general purpose high level
-interpreted programming language especially suited to algebraic
-applications. It is easy to extend GAP’s functionality; currently over
-120 GAP packages contributed by third party authors are distributed
-with each release.
-GAP is integrated into OSCAR via the [GAP.jl](https://github.com/oscar-system/GAP.jl) Julia package.
-
-The system [polymake](https://polymake.org) is a standard tool for dealing with convex
-polytopes, polyhedral fans, tropical hypersurfaces,
-and related objects from combinatorics and
-geometry. It is designed as a hybrid written in C++ and Perl. A
-sophisticated rule based mechanism decides which low-level C++
-functions are to be called to satisfy user demands.
-Polymake is integrated into OSCAR via the [Polymake.jl](https://github.com/oscar-system/Polymake.jl) Julia package.
-
-[Singular](https://www.singular.uni-kl.de) is a well-established computer algebra system for polynomial
-computations, with particular emphasis on applications in algebraic
-geometry, commutative algebra, and singularity theory, and with two
-subsystems for non-commutative algebra,
+**OSCAR** is an **O**pen **S**ource **C**omputer **A**lgebra **R**esearch System
+written in the [Julia programming language](https://julialang.org). It
+provides a unified, high-performance framework for computations in
+algebra, geometry, number theory, and polyhedral geometry. OSCAR builds
+on the capabilities of four major systems:
+  [GAP](https://www.gap-system.org),
+  [Singular](https://www.singular.uni-kl.de),
+  [Polymake](https://polymake.org), and the now-retired
+  [ANTIC](https://github.com/flintlib/antic),
```

Comment:
so after my changes this now is the "antic" link which @thofma may or may not wish to replace